src/components/Layout/Layout.module.scss

```diff
@@ -3,3 +3,24 @@
   font-family: roboto;
   max-width: 100%;
 }
+
+.header {
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
```

Comment:
We can delete default styles, like 
- `flex-direction: row`
- `wrap: nowrap`

Reply:
Sure! Removed the default CSS. Thank you! 👍